datacatalog/aws-athena-datacatalog.json

```
@@ -0,0 +1,110 @@
+            }
+        },
+        "Tag": {
+            "type": "object",
+            "properties": {
+                "Key": {
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 128
+                },
+                "Value": {
+                    "type": "string",
+                    "minLength": 0,
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "Key",
+                "Value"
+            ],
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "Name": {
+            "description": "The name of the data catalog to create. The catalog name must be unique for the AWS account and can use a maximum of 128 alphanumeric, underscore, at sign, or hyphen characters. ",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 256
+        },
+        "Description": {
+            "description": "A description of the data catalog to be created. ",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1024
+        },
+        "Parameters": {
+            "description": "Specifies the Lambda function or functions to use for creating the data catalog. This is a mapping whose values depend on the catalog type. ",
+            "type": "object",
+            "patternProperties": {
+                ".*": {
+                    "type": "string",
+                    "maxLength": 51200
+                }
+            }
+        },
+        "Tags": {
+            "description": "A list of comma separated tags to add to the data catalog that is created. ",
+            "$ref": "#/definitions/Tags"
+        },
+        "Type": {
+            "description": "The type of data catalog to create: LAMBDA for a federated catalog, GLUE for AWS Glue Catalog, or HIVE for an external hive metastore. ",
+            "type": "string",
+            "enum": [
+                "LAMBDA",
+                "GLUE",
+                "HIVE"
+            ]
+        }
+    },
+    "required": [
+        "Name",
+        "Type"
+    ],
+    "handlers": {
+        "create": {
+            "permissions": [
+                "athena:CreateDataCatalog",
+                "athena:TagResource"
+            ]
+        },
+        "read": {
+            "permissions": [
+                "athena:GetDataCatalog",
+                "athena:ListTagsForResource"
+            ]
+        },
+        "update": {
+            "permissions": [
+                "athena:UpdateDataCatalog",
+                "athena:TagResource",
+                "athena:UntagResource"
+            ]
+        },
+        "delete": {
+            "permissions": [
+                "athena:DeleteDataCatalog",
+                "athena:UntagResource"
+            ]
+        },
+        "list": {
+            "permissions": [
+                "athena:ListDataCatalog"
```

```suggestion
                "athena:ListDataCatalogs"
```
https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonathena.html